Tests/UnitTests/Plugins/Sycl/Seeding/SeedfinderSyclTest.cpp

```diff
@@ -19,6 +19,10 @@
 #include "Acts/Seeding/SpacePointGrid.hpp"
 #include "Acts/Utilities/Logger.hpp"
 
+#include "vecmem/memory/sycl/shared_memory_resource.hpp"
```

Comment:
The current implementation only uses the host and device resource. The shared one could be removed for now.

Later on we will probably complicate the whole setup for the test quite a bit further. But for now let's just include the minimal set.

Reply:
Done. I just can't close this one either. :frowning: